fix indexing of lines read from sources.conf for being able to …

lucianm opened the pull request:
… restore the order when displaying the satellite list in the diseqc configuration dialog. There, the order is that of the dictionary (an arbitrary one), because sorted(...) to a list before feeding the GUI dialog does not work when all indices are actually 1. This modification fixes this. The satllite list displayed in the scan menu was correctly sorted, but that one comes from wirbelscan via restfulapi, I guess...

lucianm committed:
fix reading lines from sources.conf in the right order, bump to 1.0.9

chewitt commented:
Can you please bump the version to 1.0.9 in addon.xml and add changelog comments - then squash commits and we will merge - then submit the matching package change to use 1.0.9 to the main repo. Thanks.

lucianm commented:
Hi, I did as you suggested, bumped version, added changelog comments and squashed everything to this same commit, now you can merge. I'm also opening a pull request for the package change rigjht now.
